snow/engine/snowman/event/queue.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package event
+
+import (
+	"context"
+
+	"github.com/ava-labs/avalanchego/utils/set"
+)
+
+type Job interface {
```

Comment:
(style) This and `Queue` require comments as top-level, exported identifiers.

[See style guide](https://google.github.io/styleguide/go/decisions#assert:~:text=All%20top%2Dlevel%20exported%20names%20must%20have%20doc%20comments)